db/migrate/20150708162823_create_fagus_item_types.rb

```diff
@@ -0,0 +1,11 @@
+class CreateFagusItemTypes < ActiveRecord::Migration
+  def change
+    create_table :fagus_item_types do |t|
+      t.string :description, :null => false
+      t.integer :is_active, :default => 0, :null => false
```

Comment:
Do you see Item types being inactive? I would see Items becoming inactive but not their types. If you see this as a feature, you may need to think about items that are under an inactive item type. 

If so